experiments/list_unsupported.sh

```diff
@@ -34,8 +34,30 @@ done
 
 echo "$1: Unsupported features\n" > "$file_name".txt
 
+export ADA_INCLUDE_PATH=/opt/gnat/lib/gcc/x86_64-pc-linux-gnu/4.9.4/rts-native/adainclude/
+if [ ! -d "$ADA_INCLUDE_PATH" ]; then
+   echo >&2 "Ada include path does not exists!"
+   exit 4
+fi
+
+PATH=/opt/gnat/bin:${PATH}
+if ! command -v gnat > /dev/null; then
+   echo >&2 "Gnat not on PATH!"
+   exit 5
+fi
+
+GPR_PROJECT_PATH=/opt/gnat/lib/gnat
```

Comment:
I'd do the same as ADA_INCLUDE_PATH here - i.e. only set this value if the user hasn't already set it.

Reply:
As above.